docusaurus.config.js

```diff
@@ -16,7 +16,7 @@ const config = {
   url: `https://${organizationName}.github.io`,
   // Set the /<baseUrl>/ pathname under which your site is served
   // For GitHub pages deployment, it is often '/<projectName>/'
-  baseUrl: `/${projectName}/`,
+  baseUrl: `/`,
```

Comment:
Why the change? If you tinker with it then the published site stops working :)

Reply:
my thinking here is that since `${projectName}/` points to `docs` the homepage would be at https://compliance-framework.github.io/docs/ with this change however, the main page would be the root i.e. https://compliance-framework.github.io/, at least that corresponds to my local deployment with yarn
![Argus_Cloud_Compliance_Framework___Argus_-_Cloud_Compliance_Framework](https://github.com/compliance-framework/docs/assets/8833750/89860b94-49dc-4ba1-8fa2-09198e661e6e)

Reply:
Unfortunately, it doesn't work like that, at least not to my knowledge. It works on localhost but fails on Github. It'd be great if you could make it work though.

Reply:
oh! Ok let me try a couple of things. I think you need to remove the change request in the PR for me to merge it